src/gradualizer_type.erl

```diff
@@ -0,0 +1,355 @@
+%% %CopyrightBegin%
```

Comment:
Delete this?

Reply:
No, don't we need to maintain license notices?

Reply:
Ah, right! This was copied from a file in OTP.